### ISSUE TITLE
Bug: Unable to get 11.0.1 working - TypeError is logged and homebridge keeps restarting

### ISSUE THREAD

**Issue by jredner** [d0n4v4nb3ck3r/homebridge-honeywell-home#583]
### Describe The Bug

Unable to get 11.0.1 working - TypeError is logged and homebridge keeps restarting (must remove plugin to restore homebridge)

### To Reproduce

Fresh install of 11.0.1, go through login/auth, restart homebridge

### Expected behavior

I expect after checking my 2 thermostats in the Honeywell Web UI config is linked.  It worked in previous versions for me no problem.

### Relevant log output

```shell
After I authorize a new App, then auth with the plugin, I select my 2 thermostats in the Honeywell Web UI, then save, and there is HomeBridge UI to close window and restart Homebridge.  After restarting, homebridge just goes in a continuous loop...



[10/23/2021, 7:41:28 PM] TypeError: Cannot read property 'debug' of undefined
    at HoneywellHomePlatform.debug (/usr/local/lib/node_modules/homebridge-honeywell-home/src/platform.ts:840:30)
    at new HoneywellHomePlatform (/usr/local/lib/node_modules/homebridge-honeywell-home/src/platform.ts:53:10)
    at /usr/local/lib/node_modules/homebridge/src/server.ts:440:40
    at Array.forEach (<anonymous>)
    at Server.loadPlatforms (/usr/local/lib/node_modules/homebridge/src/server.ts:372:27)
    at Server.start (/usr/local/lib/node_modules/homebridge/src/server.ts:159:29)
[10/23/2021, 7:41:28 PM] Got SIGTERM, shutting down Homebridge...
[10/23/2021, 7:41:29 PM] [LitterRobotPlatform] Shutdown
[10/23/2021, 7:41:29 PM] [Harmony Hub] INFO - shutdown
[10/23/2021, 7:41:29 PM] [Rachio] Loading Rachio device: Jordan Redner  - aa2ddfdf-9cc0-4a1f-85a6-daf24ec1307c
[10/23/2021, 7:41:29 PM] [Rachio] Add Device: Jordan Redner 
[10/23/2021, 7:41:30 PM] [Rachio] Adding Zone: Front Drip
[10/23/2021, 7:41:30 PM] [Rachio] Adding Zone: Front Lawn

other log lines...

[10/23/2021, 7:41:14 PM] [HB Supervisor] Homebridge Process Ended. Code: 143, Signal: null

.... repeats previous after restarting

To get my homebridge system back I remove the plugin.
```

### Config for homebridge-honeywell-home

```shell
{
            "name": "HoneywellHome",
            "credentials": {
                "consumerKey": "xxxxxxxxx",
                "consumerSecret": "xxxxxxxxxx",
                "accessToken": "xxxxxxxxx",
                "refreshToken": "xxxxxxxxx"
            },
            "platform": "HoneywellHome"
        }
```

### Screenshots

No screen images

### Device & Model

Honeywell Home RCHT8610WF2006/W, T5 Smart Thermostat, Black

### Node.js Version

v14.18.1

### NPM Version

v6.14.15

### Homebridge Version

v1.3.5

### Homebridge Honeywell Home Plugin Version

v11.0.1

### Homebridge Config UI X Plugin Version

v4.41.2

### Operating System

Raspbian

**Comment by dustindclark** [comment]
Same problem. Had to downgrade to v10.1.1.

**Comment by jredner** [comment]
Indeed 10.1.1 resolved my issue.  Thank you.

**Comment by donavanbecker** [comment]
@jredner i will look into this.

**Comment by donavanbecker** [comment]
Fixed in Version 11.1.0
